independent-projects/tools/devtools-common/src/main/java/io/quarkus/platform/tools/ToolsUtils.java

```diff
@@ -1,5 +1,6 @@
 package io.quarkus.platform.tools;
 
+import com.google.common.base.Splitter;
```

Comment:
I'd strongly advise to not use any Guava classes at all, since it increases the native binary size considerably.

Reply:
I'll fix that, thanks